src/main.py

```diff
@@ -1,6 +1,10 @@
 """ The main entry point to the script """
 
 from src.table_data_completeness import table_data_completeness
+from src.table_data_formatting import (
+    table_data_formatting_has_data_cols,
+    table_data_formatting_has_hierarchical_cols,
+)
 
 
 def main():
```

Comment:
Ignore main for the moment - I don't think we know exactly how and where this will be used yet, so I haven't done anything towards calling this nicely (yet)

Reply:
I'm anticipating this project having a few sub-modules, I've designed the two sub-modules I've written to be callable through:
```bash
python -m src.metadata
python -m src.download
```

Reply:
I think it could be really helpful to sit, or share a remote whiteboard, to write out the user and app operations so we can sketch a conceptual view of how we want the user to be able to interact with the app & how we want to structure the codebase?